bot/exts/info/information.py

```diff
@@ -19,6 +19,7 @@
 from bot.utils.checks import cooldown_with_role_bypass, has_no_roles_check, in_whitelist_check
 from bot.utils.time import TimestampFormats, discord_timestamp, humanize_delta
 
+
```

Comment:
Why was the line added here? It's inconsistent with other modules and creates an unnecessary diff